src/licensedcode/data/rules/lgpl-2.0-plus_with_libtool-exception-2.0_1.yml

```diff
@@ -1,7 +1,7 @@
 license_expression: lgpl-2.0-plus WITH libtool-exception-2.0
 is_license_notice: yes
 minimum_coverage: 90
-relevance: 100
+relevance: 90
```

Comment:
Why is this not 100% relevant? is there any ambiguity that is a `lgpl-2.0-plus WITH libtool-exception-2.0`? I do not think so, furthermore, this is a large enough rule, so you could/should eschew storing a `relevance` there IMHO

Reply:
I forgot to remove this change, i.e. after doing [this](https://github.com/nexB/scancode-toolkit/pull/1963#discussion_r391002841). Fixing this and the commit message too. Thanks for pointing out.